src/GlobalStyles.js

```diff
@@ -0,0 +1,16 @@
+import { createGlobalStyle } from 'styled-components/macro'
+
+export default createGlobalStyle`
+* {
+    box-sizing: border-box;
+    font-family: "Roboto", sans-serif;
+}
+body {
+    background: linear-gradient(180deg, rgba(0,0,29,1), rgba(0,0,11,1));
+    margin: 0;
+    padding: 0;
+    
+}
+
```

Comment:
If you are planning to use some colors multiple times in your app, you can use `:root` to declare global CSS variables (like so: https://developer.mozilla.org/en-US/docs/Web/CSS/:root) or even create separate `Color.js` file.

Reply:
This is very helpful, will implement it.